packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2276,6 +2276,29 @@ function warnAboutInvalidUpdatesOnClassComponentsInDEV(fiber) {
   }
 }
 
+export function warnIfNotCurrentlyActingEffectsInDEV(fiber: Fiber): void {
+  if (__DEV__) {
+    if (ReactShouldWarnActingUpdates.current === false) {
+      warningWithoutStack(
+        false,
+        'Your test just caused an effect from %s, but was not wrapped in act(...).\n\n' +
```

Comment:
This should be a better message, punting on that till I get approval on the rest of this.